### DISCUSSION TITLE
Which is more suitable for a cross-platform commercial project: Keras.NET or Tensorflow.NET?

### DISCUSSION THREAD

**Discussion by CodeRabbit957** [SciSharp/Keras.NET#243] [Q&A]
I'm trying to create something in Visual Studio (.NET 6.0) that I'd eventually like to commercialise and I'm not sure which would be most suitable. 
I'm not particularly knowledgeable about licensing, so I'm not sure if there are any restrictions I need to be aware of with the Apache 2.0 licence for Tensorflow.NET. 
With Keras.NET, will the dependency on Python be a problem across different platforms?

Any help would be appreciated. Thanks.

**Comment by SanftMonster** [comment]
Hi, sorry for the late reply. As far as I know, Tensorflow and Keras both uses Apache 2.0 license, so does Tensorflow.NET. And Keras.NET uses MIT license. I think license won't be a problem for commercial use.

> With Keras.NET, will the dependency on Python be a problem across different platforms?

Python and dotnet core are both cross-platform now and feel free to choose the one you like. 😊

**Comment by relentless-dev-purchases** [comment]
There is a massive amount of opportunity in food service for the application of movement and exertion analysis. That double cheeseburger someone flipped for you is really a ~50 Lb box stored in a FIFO environment as if in an ice cave for Ghengis Khan.

**Comment by zhuoshui-AI** [comment]
Why not TorchSharp ? It's offical and more maintain
